### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by mschaef-da**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,82 +25,200 @@ Once installed, verify `ddit` by launching it without arguments:
...
     release             Tag and release the current DIT file.
+    run                 Run the current project as an integration.
     show                Verify and print the current metadata file.
     targetname          Print the build target filename to stdout
 
 optional arguments:
   -h, --help            show this help message and exit
   --verbose             Turn on additional logging.
-2021-02-03T19:24:31-0500 [ERROR] (ddit) Fatal Error: Subcommand missing.
+2021-06-30T13:20:33-0500 [ERROR] (ddit) Fatal Error: Subcommand missing.
```
Comment (alexmatson-da) [inline README.md:48]: Tangential, but have we thought about leaving out this error message when no arguments are supplied?

File: README.md [hunk 2]
```diff
@@ -23,82 +25,200 @@ Once installed, verify `ddit` by launching it without arguments:
...
-For examples of what this looks like in practice, please see one of
-several sample applications available through Daml Hub:
+Available catalog fields include the following:
+
+| Field | Description |
+|-------|-------------|
+| `name` | Machine-readable name for the project. All DIT files with the same project name are considered to be different versions of the same project. |
+| `version` | The version number of the project |
+| `short_description` | A short description of the project, displayed in the App tile on the console.|
+| `description` | An optional long-fomr description of the project. |
```
Comment (alexmatson-da) [inline README.md]:
```suggestion
| `description` | An optional long-form description of the project. |
```

File: README.md [hunk 3]
```diff
@@ -23,82 +25,200 @@ Once installed, verify `ddit` by launching it without arguments:
...
+These other fields (the ones not listed above) should be considered
+deprecated for future use.
+
+## `pkg/`
+
+This is an optional directory that contains other resources to be
+includued in the output DIT. This is usually used to package the icon
+named by `icon_file`.
+
+## DAML Project
```
Comment (alexmatson-da) [inline README.md]:
```suggestion
## Daml Project
```

File: README.md [hunk 4]
```diff
@@ -23,82 +25,200 @@ Once installed, verify `ddit` by launching it without arguments:
...
+
+`ddit` is integrated into the
+[Daml Sdk]((https://docs.daml.com/getting-started/installation.html)),
```
Comment (alexmatson-da) [inline README.md]:
```suggestion
[Daml SDK]((https://docs.daml.com/getting-started/installation.html)), 
```

**Review by alexmatson-da — APPROVED** [review]
lgtm, just some readme sugggestions

File: README.md (hunk 1 above)
Comment (mschaef-da) [reply README.md:48]: I guess it's implied, but I like the explicit description of why tool isn't doing anything other than printing a help message.

**Pull request merged**